common/views/components/SearchFilters/SearchFilters.js

```diff
@@ -107,189 +99,29 @@ const SearchFilters = ({
-                <Space
-                  as="span"
-                  h={{
-                    size: 'm',
-                    properties: ['margin-right'],
-                  }}
-                >
-                  Active filters:
-                </Space>
-              </h2>
-              {productionDatesFrom && (
-                <NextLink
-                  passHref
-                  {...worksUrl({
-                    ...searchParams,
-                    page: 1,
-                    productionDatesFrom: null,
-                  })}
-                >
-                  <a>
-                    <CancelFilter text={`From ${productionDatesFrom}`} />
-                  </a>
-                </NextLink>
-              )}
-              {productionDatesTo && (
-                <NextLink
-                  passHref
-                  {...worksUrl({
-                    ...searchParams,
-                    page: 1,
-                    productionDatesTo: null,
-                  })}
-                >
-                  <a>
-                    <CancelFilter text={`To ${productionDatesTo}`} />
-                  </a>
-                </NextLink>
-              )}
+  const sharedProps = {
+    searchForm,
+    searchParams,
+    workTypeAggregations,
+    changeHandler,
+    inputDateFrom,
+    inputDateTo,
+    setInputDateFrom,
+    setInputDateTo,
+    workTypeFilters,
+    productionDatesFrom,
+    productionDatesTo,
+    workTypeInUrlArray,
+  };
 
-              {workTypeInUrlArray.map(id => {
-                const workTypeObject = workTypeFilters.find(({ data }) => {
-                  return data.id === id;
-                });
-                return (
-                  <Fragment key={id}>
-                    <NextLink
-                      key={workTypeObject.data.id}
-                      {...worksUrl({
-                        ...searchParams,
-                        workType: searchParams.workType.filter(
-                          w => w !== workTypeObject.data.id
-                        ),
-                        page: 1,
-                      })}
-                    >
-                      <a>
-                        <CancelFilter text={workTypeObject.data.label} />
-                      </a>
-                    </NextLink>
-                  </Fragment>
-                );
-              })}
-              <NextLink
-                passHref
-                {...worksUrl({
-                  ...searchParams,
-                  workType: null,
-                  page: 1,
-                  productionDatesFrom: null,
-                  productionDatesTo: null,
-                  itemsLocationsLocationType: null,
-                })}
-              >
-                <a>
-                  <CancelFilter text={'Clear all'} />
-                </a>
-              </NextLink>
-            </Space>
-          </div>
-        )}
-      </Space>
-    </div>
+  return (
+    <>
+      {isMobile ? (
```

Comment:
Hmm. Why JS and CSS like everything else for responsive layouts?

Reply:
We could create some helpers and just apply those classes on wrappers around the components.

Reply:
Having the named form elements duplicated on the page was problematic when it came to submitting the form (and the UI diverges too much between desktop and mobile to realistically be covered off by the same elements with CSS alone).

After chatting with @gestchild, reasoned swapping them out in this manner made the most sense.